setup.sh

```diff
@@ -211,6 +211,7 @@ function usage()
   say '--help      | -h     Display this usage message'
   say "--install   | -i     Install $app_name"
   say "--uninstall | -u     Uninstall $app_name"
+  say '--setup-kw-env-prompt | -s  Show current env in terminal'
```

Comment:
Now that we are not overridden the PS1 variable, I think we don't need this. We can just install directly via ./setup -i.